Signum.Engine/Json/FilterJsonConverter.cs

```diff
@@ -130,7 +130,7 @@ public class ColumnTS
 
     public Column ToColumn(QueryDescription qd, bool canAggregate)
     {
-        var queryToken = QueryUtils.Parse(token, qd, SubTokensOptions.CanElement | (canAggregate ? SubTokensOptions.CanAggregate : 0));
+        var queryToken = QueryUtils.Parse(token, qd, SubTokensOptions.CanElement | SubTokensOptions.CanOperation | (canAggregate ? SubTokensOptions.CanAggregate : 0));
```

Comment:
What I mean is canAggregate? SubTokenOptions.CanAggregate : SubTokenOptions.CanOperation